# what is the output of the following code?

def func1():
    x = 50
    return x
    
func1()
print(x)

#nameError